core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableSchemaBuilder.java

```diff
@@ -42,6 +45,16 @@
 
   private List<ColumnSchema> otherColumns = new LinkedList<>();
 
+  private int blockSize;
+
+  public TableSchemaBuilder blockSize(int blockSize) {
+    if (blockSize == 0) {
```

Comment:
Logic and prompt are inconsistent, should be '<='

Reply:
ok